src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -135,7 +135,7 @@ void RedisStoreClient::MGetValues(const std::string &table_name,
   }
 }
 
-RedisStoreClient::RedisStoreClient(std::shared_ptr<RedisClient> redis_client)
+RedisStoreClient::RedisStoreClient(std::unique_ptr<RedisClient> &&redis_client)
```

Comment:
I think `&&` is not necessary?

Reply:
Ah, I see what you mean, unique_ptr can't be copied anyway so no need